### PR TITLE
Fixed typos and updated diagram

### PR DESCRIPTION
**Pull request by isamauny**
Please review updated review and diagram.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -167,17 +168,17 @@ You should see a response similar to this. This `x-access-token` is a JWT that y
...
 
-1. Wrong verb: the opertion `Register` is defined to use `POST`, try calling it with `GET` or other verbs, and see how requests are blocked.
+1. Wrong verb: the operation `Register` is defined to use `POST`, try calling it with `GET` or other verbs, and see how requests are blocked.
 
 2. Wrong path: any request to a path _not_ defined in the OAS definition is blocked, try `/api/foo`, for example.
 
 3. Wrong `Content-Type`: the OpenAPI definition states that the operation `/api/register` requires input in the form of `application/json`. If you use a different value or if you do not specify the `Content-Type`, the request is blocked. 
 
-4. Missing a parameter that the input JSON structure requires: the schema for the operation `/api/register` specifies that the parameters `user`, `name`, `email`, and `password` are mandatory. If you do not specify all of these parameters, the requests is blocked. 
+4. Missing a parameter that the input JSON structure requires: the schema for the operation `/api/register` specifies that the parameters `user`, `name`, `email`, and `password` are mandatory. If you do not specify any of these parameters, the request is blocked.
```
Comment (lruusuvuori) [inline README.md]: "If you do not specify any of these parameters,..." This now reads that the request only gets blocked if none of the parameters have been defined, and that if even a single one is specified, the request gets through. Is this what you meant?

If not, I'd either suggest keeping the original (all parameters must be defined or the request gets blocked), or switching to something completely different, like: "If you leave out any of these parameters,..."

File: README.md [hunk 2]
```diff
@@ -167,17 +168,17 @@ You should see a response similar to this. This `x-access-token` is a JWT that y
 
 42Crunch API Firewall validates API requests and responses according to the OpenAPI definition of the protected API. You can test  the firewall behavior with the following requests:
```
Comment (lruusuvuori) [inline README.md]: Extra space between "test" and "the" in "test the firewall".

File: README.md [hunk 3]
```diff
@@ -167,17 +168,17 @@ You should see a response similar to this. This `x-access-token` is a JWT that y
...
 
 3. Wrong `Content-Type`: the OpenAPI definition states that the operation `/api/register` requires input in the form of `application/json`. If you use a different value or if you do not specify the `Content-Type`, the request is blocked. 
 
-4. Missing a parameter that the input JSON structure requires: the schema for the operation `/api/register` specifies that the parameters `user`, `name`, `email`, and `password` are mandatory. If you do not specify all of these parameters, the requests is blocked. 
+4. Missing a parameter that the input JSON structure requires: the schema for the operation `/api/register` specifies that the parameters `user`, `name`, `email`, and `password` are mandatory. If you do not specify any of these parameters, the request is blocked. 
 
-5. Wrong format for values: if you specify a value (such as email) in a format that does not match the schema, the request is blocked.
+5. Wrong format for values: if you specify a value (such as email) in a format that does not match the schema, the request is blocked. For example, try to register a user with email `user@acme.com@elysee.fr` (you can read how this was exploited by hackers here)
```
Comment (lruusuvuori) [inline README.md]: Add full stop at the end of the sentence. Also, were you planning to link to a newsletter issue here? Not sure if it's just the diff not showing the URL, or if the link is missing.

File: README.md [hunk 4]
```diff
@@ -167,17 +168,17 @@ You should see a response similar to this. This `x-access-token` is a JWT that y
...
 
-6. MongoDB injection: If you introduce a MongoDB injection like `[$ne]=1` instead of the password value when calling the login method, the request and login are blocked.
+6. MongoDB injection: The API has a MongoDB injection vulnerability which can be exploited to login to the application without specifying a password. You can try to login using the following raw parameters in Postman `user=user@acme.com&pass[$ne]=`. You will be able to login with the unprotected API, but will be blocked by our firewall.
```
Comment (lruusuvuori) [inline README.md]: "The Pixi API has a MongoDB injection vulnerability that allows logging in to the application without specifying a password. You can try this by using the raw parameters `user=user@acme.com&pass[$ne]=` in Postman for a login request. You will see that you can log in to the unprotected API, but the request is blocked by API Firewall on the protected API."

**Review by lruusuvuori — CHANGES_REQUESTED** [review]
In the image, there seems to be an extra space in "42 Crunch Platform" at the top box, should be "42Crunch Platform".

Also, the red lines are maybe not the best option for redacting info. They pull the focus to themselves rather than blending to the background, so that all you notice in that image are the red lines. I would just blur the text in the screenshot, mock-up attached:

Deployment_detail

File: README.md (hunk 1 above)
Comment (isamauny) [reply README.md]: It's the latter : if one or multiple mandatory parameters are missing the request is blocked.

**Review by lruusuvuori — APPROVED** [review]

**Pull request merged**